news/9910.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix for ValueError('ZIP does not support timestamps before 1980') on GitHub package install.
```

Comment:
```suggestion
Allow ZIP to archive files with timestamps earlier than 1980.
```

Describe the thing that changed, not the thing it fixed, since the solution is not scoped to fix that *exact* thing.